Day7n8n9/github-api-project-ts/src/components/SelectByCity/SelectByCity.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { Dispatch, SetStateAction } from "react";
+
+import data from "../../utils/state-city-data.json";
+import classes from "./selectByCity.styles.module.css";
+
+type Props = {
+  selectedCity: string;
+  setSelectedCity: Dispatch<SetStateAction<string>>;
```

Comment:
not sure why you use Dispatch, setStateAction

Reply:
These are generic types provided by react while writing React-TypeScript code.